### DISCUSSION TITLE
Artist/album folder name and track names

### DISCUSSION THREAD

**Discussion by steve1977** [RandomNinjaAtk/docker-lidarr-extended#42] [General]
Does the script follow the naming convention in lidarr? I.e., what is shown under Standard Track Format in media management?

Would be interested to understand what you guys use as naming convention and what works best to avoid in combo with Plex (if relevant).

**Comment by Protektor-Desura** [comment]
the scripts trigger a input or upload to Lidarr to go, hey here is this file, process it. It eventually gets renamed by Lidarr during processing based on what you set Lidarr to do.

**Comment by steve1977** [comment]
Cool, thanks! So, it will be consistent across script and usenet.

You using the default rename settings?

**Reply by Protektor-Desura** [reply]
I have a custom rename I use in Lidarr. You can use the default renaming if you want.
